### PR TITLE
Blog posts view on homepage

### PR DESCRIPTION
**Pull request by PrimozGodec**

### CODE REVIEW

**Comment by PrimozGodec** [comment]
I finished it on friay. May it be checked before I leave (4. 10.)?

**Comment by kernc** [comment]
@astaric remind me to ask you something about CSS? :smiley:

File: homepage/templatetags/tag_extras.py [hunk 1]
```diff
@@ -43,6 +60,27 @@ def grab_feed_all():
...
 
+def cut_string(string, post_length):
+    if len(string) < post_length:
+        return string
+    else:
+        # find fist space before limit
+        for i in range(post_length, 0, -1):
+            if string[i] == ' ':
+                break
+        return string[:i]
```
Comment (kernc) [inline homepage/templatetags/tag_extras.py]:
```py
else:
    string = string[:post_length]
    return string[:string.rfind(' ')]
```

**Comment by ajdapretnar** [comment]
Am I the only one seeing this?

blog

If not, pictures need to be resized to fixed width (I guess this is Kernc's suggestion) and also we talked about a grey-brown panel for blog posts background. Plus the <span sth sth> or whatever is causing bold font needs to be removed.

**Comment by PrimozGodec** [comment]
It is totaly different to what I did and see. I will check what is wrong. Otherwise I will come in lab on friday and we can check together.

File: homepage/templatetags/tag_extras.py (hunk 1 above)
Comment (ajdapretnar) [inline homepage/templatetags/tag_extras.py]: Disagree.

**Pull request merged**